substratevm/src/com.oracle.svm.driver/resources/container-default/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=container-registry.oracle.com/os/oraclelinux:8-slim
```

Comment:
We also need a GPL license header for the two Dockerfiles.
For adding comments in Dockerfiles see https://docs.docker.com/engine/reference/builder/#format

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/e03950a15877b5739343115cf6c031496b3ae976